packages/xo-server/src/utils.js.flow

```diff
@@ -13,3 +13,5 @@ declare export function serializeError(error: Error): Object
 declare export function streamToBuffer(stream: Readable): Promise<Buffer>
 
 export type SimpleIdPattern = {| id: string | {| __or: string[] |}, |}
+
+export type ReportWhen = 'always' | 'failure' | 'never'
```

Comment:
Why in `utils`?

Reply:
I don't see other place to declare it.

Reply:
You have two choices:

- create a new module
- don't mutualize this type: we're going to abandon Flow anyway